### ISSUE TITLE
Problem with Network icon on Desktop from Linux Mint 20.0

### ISSUE THREAD

**Issue by DorianB2017** [linuxmint/linuxmint#278]
```
 * Cinnamon version (cinnamon --version) - 4.6.6
   - Please specify if you are using the daily builds PPA
     (https://launchpad.net/~linuxmint-daily-build-team/+archive/ubuntu/daily-builds). - No
 * Distribution - (Mint 17.2, Arch, Fedora 25, etc...) - Mint 20.0
 * Graphics hardware *and* driver used 
 * 32 or 64 bit - 64 bit
```
**Issue**
After upgrading from Linux Mint v.19.3 - Cinnamon to Linux Mint v.20.0 - Cinnamon, the "Network" icon when accessed, it requires association with a program even if I reactivate the option in "System settings" > "Desktop", such as can be seen in the attached images.

I'm currently using the "Numix icons theme" theme. (The same error is when using the "Mint-Y" theme, for example.)

![Captură de ecran din 2020-07-12 16-32-48a](https://user-images.githubusercontent.com/30018486/87248306-701c5c00-c461-11ea-9416-c5800e552327.png)
![Captură de ecran din 2020-07-12 16-38-02a](https://user-images.githubusercontent.com/30018486/87248340-90e4b180-c461-11ea-8819-c9be084832e4.png)
![Captură de ecran din 2020-07-12 16-43-27](https://user-images.githubusercontent.com/30018486/87248352-a3f78180-c461-11ea-9644-3987fa3fa50c.png)

**Comment by xenopeek** [comment]
Can you confirm after running `update-desktop-database` your issue was solved?

I think the screenshot is unrelated. It shows files from the app-install-data package which is by Software Sources to display programs that you can install.

**Comment by DorianB2017** [comment]
No. Not. Now, after the reset, the same message associated with a program and now does not want to open the "Network" window. Now, after resetting (or turn on), ```update-desktop-database``` = zero effect. When I initially executed this command, it worked.

It's a bug for upgrade to Linux Mint v. 20.0 non-English (at least for Linux Mint v. 20 in Romanian).

**Comment by DorianB2017** [comment]
> What does this command return?
> 
> ```
> xdg-mime query default x-scheme-handler/network
> ```
For leigh123linux

Nothing happens. I wrote above what happens, regardless of whether I give this command or the other written above, the same message of association with a program.
